bot/resources/evergreen/trivia_quiz.json

```diff
@@ -604,5 +604,309 @@
+    },
+    {
+      "id": 430,
+      "question": "Which of these languages is the youngest? Lisp, Python, Java, Haskell, Prolog, Ruby, Perl",
+      "answer": "Java"
+    }
+  ],
+  "python": [
+    {
+      "id": 501,
+      "question": "Is it all objects (y/n)?",
+      "answer": "y, yes"
+    },
+    {
+      "id": 502,
+      "question": "Name the only non-dunder method of the builtin slice object.",
+      "answer": "indices"
+    },
+    {
+      "id": 503,
+      "question": "What exception, other than `StopIteration`, can you raise from a `__getitem__` dunder to indicate to an iterator that it should stop?",
+      "answer": "IndexError"
+    },
+    {
+      "id": 504,
+      "question": "What type does the `&` operator return given 2 `dict_keys` objects?",
+      "answer": "set"
+    },
+    {
+      "id": 505,
+      "question": "Can you pickle a running `list_iterator` (y/n)?",
+      "answer": "y, yes"
+    },
+    {
+      "id": 506,
+      "question": "What attribute of a closure contains the value closed over?",
+      "answer": "cell_contents"
+    },
+    {
+      "id": 507,
+      "question": "What name does a lambda function have?",
+      "answer": "<lambda>"
+    },
+    {
+      "id": 508,
+      "question": "Which file contains all special site builtins, such as help or credits?",
+      "answer": "_sitebuiltins"
+    },
+    {
+      "id": 509,
+      "question": "Which module when imported opens up a web browser tab that points to the classic 353 XKCD comic mentioning Python?",
+      "answer": "antigravity"
+    },
+    {
+      "id": 510,
+      "question": "Which attribute is the documentation string of a function/method/class stored in (answer should be enclosed in backticks!)?",
+      "answer": "`__doc__`"
+    },
+    {
+      "id": 511,
+      "question": "What is the official name of this operator `:=`, introduced in 3.8?",
+      "answer": "assignment-expression operator"
+    },
+    {
+      "id": 512,
+      "question": "When was Python first released?",
+      "answer": "1991"
+    },
+    {
+      "id": 513,
+      "question": "Where does the name Python come from?",
+      "answer": "Monty Python, Monty Python's Flying Circus"
+    },
+    {
+      "id": 514,
+      "question": "How is infinity represented in Python?",
+      "answer": "float(\"infinity\"), float('infinity'), float(\"inf\"), float('inf')"
+    },
+    {
+      "id": 515,
+      "question": "Which of these characters is valid python outside of string literals in some context? `@`, `$`, `?`",
+      "answer": "@"
+    },
+    {
+      "id": 516,
+      "question": "Which standard library module is designed for making simple parsers for languages like shell, as well as safe quoting of strings for use in a shell?",
+      "answer": "shlex"
+    },
+    {
+      "id": 517,
+      "question": "Which one of these protocols/abstract base classes does the builtin `range` object NOT implement? `Sequence`, `Iterable`, `Generator`",
```

Comment:
Maybe keep consistency here and enclose multiple choice options in parentheses or do the inverse

Reply:
there is consistency - none of these non-lettered ones are enclosed in parentheses